config/secrets.yml

```diff
@@ -17,10 +17,21 @@
 
 # Environmental secrets are only available for that specific environment.
 
+default: &default
```

Comment:
# RTFM

Reply:
The `&default` is the old way of doing things, if you read the top of this file, it shows you how to do it in the new style

Reply:
Sorry @spencerldixon I just looked at Protected Planet to learn how to do this. I will Read The Friendly Manual ;-)